### PR TITLE
AOM-103: Write test for Help Component

### PR DESCRIPTION
**Pull request by Charpell**
### SUMMARY:
Presently, there is no test for the newly created Help Component. This ticket creates a unit test

### CODE REVIEW

File: app/js/components/manageApps/ManageApps.jsx [hunk 1]
```diff
@@ -859,14 +859,14 @@ export default class ManageApps extends React.Component {
               <span className="pull-right manage-settings-wrapper">
                 <span id="startall-modules-btn"
                   className="btn btn-secondary"
-                  onClick={updatesAvailable? this.clearUpdates : this.checkForUpdates}>{updatesAvailable? 'Back to all Addons': 'Check For Updates'}</span>
+                  onClick={updatesAvailable? this.clearUpdates : this.checkForUpdates}>{updatesAvailable? 'Back to all Addons': 'Check for Updates'}</span>
```
Comment (brayoh) [inline app/js/components/manageApps/ManageApps.jsx]: you forgot to add a space between the variable and the question mark

**Comment by aenkya** [comment]
@Charpell What's the file coverage

**Comment by Charpell** [comment]
@Elbertbiggs360 The coverage is 100%

File: app/js/components/manageApps/ManageApps.jsx (hunk 1 above)
Comment (Charpell) [reply app/js/components/manageApps/ManageApps.jsx]: @brayoh Thank you, I have updated the PR

**Review by aenkya — COMMENTED** [review]
Looks good to me

**Pull request merged**